dss/stepfunctions/gscopyclient/implementation.py

```diff
@@ -23,12 +22,18 @@ class _Key:
     TOKEN = "token"
 
 
+def get_gcp_client():
+    # TODO: (ttung) remove this once Config.get_cloud_specific_handles is refactored.
```

Comment:
Unless you have a PR in the works already, this is bound to fall through the cracks. What I do cases like this create a ticket and refer to the ticket in the TODO.